### PR TITLE
Mobile view fixes

### PR DESCRIPTION
**Pull request by ulshv**

### CODE REVIEW

File: src/app/modules/dsp/components/ServiceTable.tsx [hunk 1]
```diff
@@ -19,7 +19,7 @@ type Props = {
 };
 
 
-const columnWidths = [5, 25, 25, 25, 20];
+const columnWidths = [5, 33, 33, 33, 33];
```
Comment (cmichel21) [inline src/app/modules/dsp/components/ServiceTable.tsx]: shouldn't these add up to 100?

Comment (ulshv) [reply src/app/modules/dsp/components/ServiceTable.tsx]: @cmichel21 for some wierd reason, when using 100% in total on Services table, it doesn't really fill all the table width (100% of container). Althoug DSPs tab looks just fine with total `columnWidths` of 100%. 

Screenshot 2019-06-22 at 16 58 11
Screenshot 2019-06-22 at 16 58 04

Comment (ulshv) [reply src/app/modules/dsp/components/ServiceTable.tsx]: So using more than 100% in total for `columnWidths` for Services table solved the issue.. Investigated good amount of time to find a better solution but couldn't...

Comment (cmichel21) [reply src/app/modules/dsp/components/ServiceTable.tsx]: Ok

**Pull request merged**